platform/mellanox/rules.mk

```diff
@@ -1,3 +1,8 @@
+# Set KVERSION correctly depending on build environment
+ifeq ($(BLDENV), buster)
+KVERSION = 5.10.0-0.bpo.8-amd64
```

Comment:
Yes, its because we need to have MFT in syncd which means we need to build MFT against a 5.10 kernel in the buster environment. 

As far as  I can tell there is no way whatsoever to have the sonic-linux-kernel artifacts available for this build so what we do is build it against the 5.10 backport kernel which seems to be close enough to work for us since MFT doesn't depend on the SDK or HW-MGMT.